src/os/windows.rs

```diff
@@ -37,13 +37,21 @@ impl WindowBuilderExt for WindowBuilder {
 
 /// Additional methods on `MonitorId` that are specific to Windows.
 pub trait MonitorIdExt {
-    /// Returns the name of the monitor specific to the Win32 API.
+    /// Returns the name of the monitor adapter specific to the Win32 API.
     fn native_id(&self) -> String;
+
+    /// Returns the handle of the monitor specific to the Win32 API.
+    fn hmonitor(&self) -> winapi::HMONITOR;
```

Comment:
I don't like exposing types of `winapi`. Please return a `*mut c_void` (from `std::os::raw::c_void`) instead, and mention in the comment that this is a `HMONITOR`.

Reply:
But `WindowExt::get_hwnd()` uses `libc::c_void`

Reply:
That's a mistake, but that requires a breaking change to fix. You can change it if you want, or just leave it and I'll change it in another PR.